src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -135,18 +135,26 @@ def consume_captions(caption_list: list) -> dict:
     for caption in caption_list:
         if 'caption_id' not in caption:
             raise ValueError('A caption ID must be set for each caption')
+
+		# Setting the default background color to white if there is no background color passed in
+        foreground_color = 'white'
+		# Setting the default test to be not underlined if no underline variable is passed in
+        underlined = False
+        if 'foreground_color' in caption and 'color' in caption['foreground_color']:
+            caption_color = caption['foreground_color']['color']
```

Comment:
`foreground_color` needs reassignment here if we ever want something other than white text

Reply:
Woops, thank you for that. I changed it from background color to foreground color because I was testing it with that, good catch.